src/components/Title/Title.test.jsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Title from "./Title";
+
+describe('Testing Title component', () => {
```

Comment:
This is also a minor thing, in the describe title I see that you have a variation of titles, sometimes you have a title like `Testing CardList`, or `NavBar` or `Testing Title component`, you can use a format to put the title in your tests to be consistent, when a project grows it helps in the long run.

You can always use a title like `{name what you are testing} {type of what you are testing, i.e. component, or hook, or context provider, or a utility function, etc}`, for example:

- `CardList Component`
- `NavBar Component`
- `Title Component`
- `useTheme Hook`
- `Theme Provider`
- etc